src/bosh_azure_cpi/lib/cloud/azure/restapi/azure_client.rb

```diff
@@ -108,8 +108,15 @@ class AzureClient # rubocop:todo Metrics/ClassLength
 
     def initialize(azure_config, logger)
       @logger = logger
-
       @azure_config = azure_config
+
+      # Allow Net::HTTP to support instance variable names via ssl_options
+      (Net::HTTP::SSL_IVNAMES << :@ssl_options).uniq!
+      (Net::HTTP::SSL_ATTRIBUTES << :options).uniq!
```

Comment:
Do we need this options in the SSL_ATTRIBUTES? I cannot see the usage of it.

Reply:
I was also of the opinion that we wouldn’t need it, but in the dev testing it turned out we do

Reply:
Hi @anshrupani, also I double checked it and can prove that it is required.